Frameworks/AddressBook/ABRecord.mm

```diff
@@ -47,12 +83,117 @@ bool ABRecordSetValue(ABRecordRef record, ABPropertyID property, CFTypeRef value
 }
 
 /**
- @Status Stub
- @Notes
+ @Status Caveat
+ @Notes Properties that return a Multi-Value are not yet supported, and a few properties
+        (alternate birthday, creation/modification dates, person kind, phonetic middle name)
+        don't have direct equivalents.
 */
 CFTypeRef ABRecordCopyValue(ABRecordRef record, ABPropertyID property) {
```

Comment:
>ABPropertyID property [](start = 48, length = 21)

This looks like it's typedef'd to int32_t, you ought to be able to do a switch statement here?

Also, you may want to change the name since it's a bit close to @property.